source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,357 @@
+#. Decompress the downloaded image.
 
-   - On Windows you can use `7zip`_ to extract the file by right-clicking the 
+   - On Windows you can use `7zip`_ to extract the file by right-clicking the
      file to *Extract Here* (in the same directory)
 
-       .. image:: ./figures/vbox/vbox-extract-cl-ISO.png
-          :alt: 7zip extract here command
+   .. figure:: figures/vbox/virtualbox-cl-installer-01.png
+      :scale: 100%
+      :alt: 7zip extract here command
+
+      Figure 1: 7zip extract here command
 
    - On Linux :
 
      .. code-block:: bash
 
-         xz -d clear-<VERSION>-installer.iso.xz
-
-
-
-
-#. The originally downloaded compressed archive file 
-   (:file:`clear-<VERSION>-installer.iso.xz`) can now be deleted.
+        xz -d clear-<VERSION>-live-server.iso.xz
 
+#. Delete the originally downloaded compressed file.
 
 Create a new |VB| virtual machine
 *********************************
 
-A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| for |CL| to be installed onto. 
-
-General instructions for creating a virtual machine and details about using 
-different settings are available on the 
+A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| where |CL|
+will be installed. General instructions for creating a virtual machine and
+details about using different settings are available on the
 `VirtualBox manual section on Creating a VM`_.
 
-
 #. Launch the |VBM| from your host system.
 
+#. Click the *New* button to create a new VM.
+
+#. Choose :guilabel:`Expert mode`.
+
+#. In :guilabel:`Create Virtual Machine`, enter the following settings:
+
+   - **Name**: Choose name (e.g. ClearLinuxOS-VM).
+   - **Type**: Linux
+   - **Version**: **Linux 2.6 / 3.x / 4.x (64-bit)**
+   - **Hard disk**: `Create a virtual hard disk now`
+   - **Memory size default**: 2048 MB (Adjust appropriately.)
+
+     .. note::
+        Later, if you want to change the amonunt of RAM allocated, power down your VM. Return to :file:`Settings > System` and change
+        :guilabel:`Base Memory` to the desired size.
 
-#. Click the *New* button to create a new VM. 
+   .. figure:: figures/vbox/virtualbox-cl-installer-02.png
+      :scale: 100%
+      :alt: Create Virtual Machine
 
-   .. image:: ./figures/vbox/vbox-new-vm.png
-      :alt: Create a new VM in VirtualBox
+      Figure 2: Create Virtual Machine
 
+#. Select :guilabel:`Create`.
 
-#. A *Create Virtual Machine* window will appear. 
-   Select the following settings:
-   
-   - Type: **Linux**
-   - Version: **Linux 2.6 / 3.x / 4.x (64-bit)**
-   - Memory size: **1024 MB** (this can be adjusted appropriately)
-   - Hard disk: **Create a virtual hard disk now**
+#. In :guilabel:`Create Virtual Hard Disk`, select:
 
-   .. image:: ./figures/vbox/vbox-create-vm-new-disk.png
-      :alt: Create a new VM in VirtualBox with a new disk
+   - **File location**
+   - **File size**: **8.00 GB**. Adjust size to your needs.
```

Comment:
The earlier purposed idea to bump this up to 32GB is a good one even for the server install. 8GB fills up fast.